cli/kata-env.go

```diff
@@ -34,6 +34,12 @@ type MetaInfo struct {
 	Version string
 }
 
+type VersionInfo struct {
+	Version string
```

Comment:
This is a little confusing - what *is* `Version`? I think it's the raw output of `$cmd --version` so I'd make that clearer by adding a comment or renaming the field. But again, I'd look at the `semver` package to avoid "re-inventing the wheel" - you could just store a `semver.Version` as that contains everything we'd need.

Reply:
Ya, same question, what is the `Version` here?  Shouldnt this be `Major`, `Minor` and `Patch` instead? 
Are we not comparing the "Patch" version at all?